src/Tags.ts

```diff
@@ -4186,6 +4186,8 @@ export interface MakerNotesTags {
   /** ☆☆☆☆ ✔ Example: "n/a" */
   SoftSkinEffect?: string
   /** ☆☆☆☆ ✔ Example: "2023:05:24 15:18:25" */
+  SonyDateTime2?: ExifDateTime | string
```

Comment:
Unfortunately this file is automatically generated, and this diff (which is required for the other diff to compile) will get reverted.

There's a list of "required" tags in mktags.ts that needs this new tag name added to.

Reply:
Okay, I added it to the mktags.ts file. I also removed it from `src/Tags.ts`, since it should be automatically generated. To rerun the generation, I saw that you need to run `yarn run mktags *picture_dir*`. Only I don't see which directory of pictures is used to generate the current list.